### PR TITLE
Release notes for Tizen Studio 3.5 release

### PR DESCRIPTION
**Pull request by vivek-ellur**
Signed-off-by: Vivek Ellur <vivek.ellur@samsung.com>

### Change Description ###

Describe your changes here.

For instance,
 - Fix broken links of native application docs.
 -  ...

### Bugs Fixed ###

Provide links to bugs here

For instance,

 - Issue #265

### API Changes ###

In case of ACR, put the link of ACR

For instance,

 - ACR: ACR-1120

### CODE REVIEW

File: docs/application/tizen-studio/release-notes/3-5-release-notes.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
+# Tizen Studio 3.5 Release Notes
+
+-   Release Date: Oct 30, 2019
+
+
```
Comment (Reni-90) [inline docs/application/tizen-studio/release-notes/3-5-release-notes.md]: Please remove the extra spacing.

Comment (vivek-ellur) [reply docs/application/tizen-studio/release-notes/3-5-release-notes.md]: Fixed

File: docs/application/tizen-studio/release-notes/3-5-release-notes.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+### New Features
+-   IDE
+	-   Java 12 Oracle/Open JDK support has been added to make the IDE and tools work with Java 12.
+	-   Template Creation Project Wizard has been refined to add Flat View which provides a different view for the purpose of project creation.
+	-   Support for Add-On type of application has been provided, its a new template for web applications from 5.5.
+	-   Support for Component Based application has been provided, its a new template for applications from 5.5.
+
+### Fixed Bugs
+-   IDE
+	-   The issue of the Samsung Certificate unable to sign application while launching application on 5.0 Emulator has been fixed.
```
[the remark this answers is not in the document]
Comment (vivek-ellur) [reply docs/application/tizen-studio/release-notes/3-5-release-notes.md]: Fixed.

File: docs/application/tizen-studio/release-notes/3-5-release-notes.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+
+### Deprecated
+-   IDE
+	-   Java 9 and Open JDK 10 support has been Deprecated.
```
[the remark this answers is not in the document]
Comment (vivek-ellur) [reply docs/application/tizen-studio/release-notes/3-5-release-notes.md]: Fixed.

File: docs/application/tizen-studio/release-notes/3-5-release-notes.md [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+
+
+## IDE and Tools
+
+### New Features
+-   IDE
+	-   Java 12 Oracle/Open JDK support has been added to make the IDE and tools work with Java 12.
+	-   Template Creation Project Wizard has been refined to add Flat View which provides a different view for the purpose of project creation.
+	-   Support for Add-On type of application has been provided, its a new template for web applications from 5.5.
+	-   Support for Component Based application has been provided, its a new template for applications from 5.5.
```
[the remark this answers is not in the document]
Comment (vivek-ellur) [reply docs/application/tizen-studio/release-notes/3-5-release-notes.md]: Fixed.

File: docs/application/tizen-studio/release-notes/3-5-release-notes.md [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+
+-   Release Date: Oct 30, 2019
+
+
+## IDE and Tools
+
+### New Features
+-   IDE
+	-   Java 12 Oracle/Open JDK support has been added to make the IDE and tools work with Java 12.
+	-   Template Creation Project Wizard has been refined to add Flat View which provides a different view for the purpose of project creation.
```
[the remark this answers is not in the document]
Comment (vivek-ellur) [reply docs/application/tizen-studio/release-notes/3-5-release-notes.md]: Fixed.

File: docs/application/tizen-studio/release-notes/3-5-release-notes.md [hunk 6]
```diff
@@ -0,0 +1,72 @@
...
+
+-  Release Date: Oct 30, 2019
+
+
+## IDE and Tools
+
+### New Features
+-   IDE
+	-   Oracle JDK/OpenJDK 12 support has been added to make the IDE and tools work with Java 12.
+	-   Project Creation Wizard has been improved to add Flat View. Flat View provides a different view while creating a project.
```
Comment (Reni-90) [inline docs/application/tizen-studio/release-notes/3-5-release-notes.md]: Project Creation Wizard is not a proper noun, so can we change "Project Creation Wizard" to "Project creation wizard"?
If not then can we use the rephrase from the [previous comment](https://github.com/Samsung/tizen-docs/pull/845#discussion_r329434941)?

Please verify.

File: docs/application/tizen-studio/release-notes/3-5-release-notes.md [hunk 7]
```diff
@@ -0,0 +1,72 @@
...
+	-   Support for the Add-On type application has been added. Add-On type is a new template for Web applications from Tizen 5.5.
+	-   Support for the component-based application has been added. Component-based application is a new template for Tizen applications from Tizen 5.5.
+
+### Fixed Bugs
+-   IDE
+	-   The issue of Samsung Certificate unable to sign application while launching an application on Tizen 5.0 Emulator has been fixed.
+
+### Deprecated
+-   IDE
+	-   Java 9 and OpenJDK 10 support has been Deprecated.
```
[the remark this answers is not in the document]
Comment (vivek-ellur) [reply docs/application/tizen-studio/release-notes/3-5-release-notes.md]: Done.

File: docs/application/tizen-studio/release-notes/3-5-release-notes.md [hunk 8]
```diff
@@ -0,0 +1,72 @@
...
+	-   Log Viewer has been Deprecated.
```
[the remark this answers is not in the document]
Comment (vivek-ellur) [reply docs/application/tizen-studio/release-notes/3-5-release-notes.md]: Done.

File: docs/application/tizen-studio/release-notes/3-5-release-notes.md [hunk 9]
```diff
@@ -0,0 +1,72 @@
+# Tizen Studio 3.5 Release Notes
+
+-  Release Date: Oct 30, 2019
+
+
+## IDE and Tools
+
+### New Features
+-   IDE
+	-   Oracle JDK/OpenJDK 12 support has been added to make the IDE and tools work with Java 12.
```
[the remark this answers is not in the document]
Comment (vivek-ellur) [reply docs/application/tizen-studio/release-notes/3-5-release-notes.md]: Done.

File: docs/application/tizen-studio/release-notes/3-5-release-notes.md (hunk 6 above)
Comment (vivek-ellur) [reply docs/application/tizen-studio/release-notes/3-5-release-notes.md]: Done.

**Comment by mijin85cho** [comment]
Thanks for the update and review.
I'll merge this PR near Oct 30, the actual release target date.

**Comment by vivek-ellur** [comment]
Release doc updated in new PR.

**Comment by mijin85cho** [comment]
Hello, @vivek-ellur 
Is there any reason that you don't update this PR and close it instead?

As it is not merged yet you can update it if you need to do so.

**Pull request closed without merging**